statusboard/src/components/ProjectsTable/ProjectsTable.scss

```diff
@@ -54,6 +54,22 @@
     }
   }
 
+  
+ }
+
+
+@media screen and (max-width:800px) {
+  th:nth-of-type(2){
+    width:calc(100% - 0.5em);
+  }
+  .hide3 {
+    th:nth-of-type(3){
+      display:none;
+    }
```

Comment:
This is only concerning the `Projects` page? Doesn't it belong in `statusboard/src/pages/Projects/Projects.scss`?

Reply:
Projects.scss was an empty file but yes best to keep things organized, so I moved this there.